test/stripe/tokens_test.exs

```diff
@@ -77,7 +77,7 @@ defmodule Stripe.TokensTest do
       {:ok, res} ->
         #Apex.ap res
         assert res.id
-        assert res.status == "paid"
+        assert res.status == "succeeded"
```

Comment:
LOL. let's not to fix that typo, so when I feel depressed, I can come back and laugh..  btw, thanks for the fast response